runtime/runtime.go

```diff
@@ -117,7 +117,7 @@ func NewRuntime(config Configuration) {
 			log.Error(err.Error())
 			panic(err.Error())
 		}
-		log.Infof("OpenShift version: %v", kubeServerInfo)
+		log.Debugf("OpenShift version: %v", kubeServerInfo)
```

Comment:
This is useful in the broker logs at info level. Now we'll have to turn on debug to see this useful information. Feels like we're making these changes when maybe bundle-lib should be configured to use a different loglevel.